src/common/config.ts

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+/**
+ * This global config object allows developers to change common settings.
+ */
+const globalConfig: { maxJsonSize: number | null | undefined } = {
+  maxJsonSize: 10 * 1024 * 1024,
+};
+
+export function setMaxJsonSize(size: number | null | undefined): void {
+  if (size != null && (!Number.isInteger(size) || size <= 0)) {
+    throw new Error("setMaxJsonSize: size must be a positive integer.");
+  }
+  globalConfig.maxJsonSize = size;
+}
+
+export function getMaxJsonSize(): number | null | undefined {
+  return globalConfig.maxJsonSize;
+}
+
+export function checkResponseSize(response: Response) {
```

Comment:
Can you add the TSDoc `@hidden` tag? This is exported, so it will be picked up by TSDoc, but we want it to be internal.